GVRf/Framework/src/org/gearvrf/GVRHybridObject.java

```diff
@@ -35,7 +30,7 @@
  * Descendant classes all have native (JNI) implementations; this base class
  * manages the native lifecycles.
  */
-public abstract class GVRHybridObject implements Closeable {
+public abstract class GVRHybridObject {
```

Comment:
Not to mention that "close" is not representative of what exactly happens; not to mention that after closing we need state checks as you can still call methods, etc..